source/docs/network_filter_fuzzing.md

```diff
@@ -0,0 +1,116 @@
+# Generic network-level filter fuzzers overview
```

Comment:
suggestion: known issues in the fuzzer maybe? e.g. state -- or things devs should keep in mind about keeping track of state

Reply:
> suggestion: known issues in the fuzzer maybe? e.g. state -- or things devs should keep in mind about keeping track of state

I added a section `known issues in the fuzzers` at the end of the document. Thanks!